Site/package.json

```diff
@@ -17,5 +17,10 @@
     "react-dom": "16",
     "@types/react": "16",
     "@types/react-dom": "16"
+  },
+  "devDependencies": {
```

Comment:
For some reason, "[borrowing](https://github.com/VEuPathDB/ClinEpiWebsite/pull/11#issue-1044731713)" @veupathdb/study-data-access from `EbrcWebsiteCommon` wasn't working for me - when running `wb site`, I was getting various errors of the form

```
TS4090: Conflicting definitions for 'zzz' found at 'xxx' and 'yyy'. Consider installing a specific version of this library to resolve the conflict.
```

(`zzz` being `@types/react`, `@types/jquery`, and `@types/jqueryui`.)

Adding `@veupathdb/study-data-access` as a dev dependency, along with the "usual" dev dependencies of `"bubleify": "^2.0.0"` and `"ify-loader": "^1.1.0"` resolved the error.

@dmfalke - do you see any issues with this approach, or better yet, have a cleaner one in mind?

Reply:
That's odd. Is there a possibility you are running a custom version of node? What does `which node` return for you?

My only concern is that this version could get out-of-sync with the version used by EbrcWebsiteCommon.